website/docs/components/accordion/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Accordion
+description: An accordion allow users to expand or collapse multiple sections of content in a stack.
+caption: Allow users to display multiple sections of content in a stack.
+status: released
+links:
+  figma: {link to the "Accordion" page in the components Figma library}
+  github: {link to the "Accordion" component's folder in the GitHub repo}
+previewImage: assets/illustrations/components/accordion.jpg
+navigation:
+  hidden: false
+  keywords: ['toggle', 'details', 'reveal', 'expand', 'collapse']
+---
+
+<section data-tab="Guidelines">
+  @include "partials/guidelines/overview.md"
+  @include "partials/guidelines/guidelines.md"
+</section>
+
+<section data-tab="Code">
+  @include "partials/code/how-to-use.md"
+  @include "partials/code/component-api.md"
+  <!-- @include "partials/code/showcase.md" -->
```

Comment:
You can probably just remove this for now, it probably makes sense to remove it from the blueprint since we've moved towards the external showcase.

Reply:
Could you tell me what these lines are for?

Reply:
It's a commented out line to include the showcase that still gets generated even though we decided to make the Showcase a separate thing for now. So it's safe to delete that line although leaving it in really doesn't hurt either.

Reply:
@KristinLBradley can you remove this on your branch if you consider necessary? It could be after I merge my branch.